pkg/local_workflows/testdata/sarif-snyk-goof-ignores.json

```diff
@@ -2066,8 +2066,7 @@
               "severity": "high",
               "originalSeverity": "low"
             }
-          },
-          "suppressions": null
+          }
```

Comment:
Question: Why was this removed? I think I'm missing some context.

Reply:
Did you double check that this is not a the way the API returns the data, just saying, it could well be that the data from the API looks like this and by removing the `null` now, we might miss a problem in the future.

Reply:
Great, thank you for sharing. Let's undo the change for now please, as this is an unrelated change.